toffy/watcher_callbacks_test.py

```diff
@@ -52,9 +54,10 @@ def test_build_fov_callback(callbacks, kwargs, data_path):
             check_mph_dir_structure(qc_dir, plot_dir, point_names)
 
 
+@patch('toffy.watcher_callbacks.visualize_qc_metrics', side_effect=mock_visualize_qc_metrics)
```

Comment:
Is there a reason why `mock_visualize_mph` isn't called here too?

Reply:
No reason, beyond I forgot; the test is still pretty fast w/o it.